src/api.ts

```diff
@@ -119,9 +119,9 @@ export class UNSCLIAPI {
             .catch(e => {
```

Comment:
Duplicated error handling, which may be resuse again later.
Please extract an error handler here.

```
.catch(customErrorHandler);
```